### DISCUSSION TITLE
How to get the test results in JSON format?

### DISCUSSION THREAD

**Discussion by Coder2023Coder** [pytest-dev/pytest-html#665] [Q&A] [answered]
How do I get the test results in JSON format in a Python program? I am not using the command line.

**Comment by RonnyPfannschmidt** [comment, marked as answer]
without way more context this question makes no sense for this plugin as its not even concerned with json results

https://github.com/pytest-dev/pytest-reportlog will output json-lines based reports/results

**Reply by Coder2023Coder** [reply]
`--report-log=FILE` in `.ini` file works and the result is what I was looking for. Thanks.

**Reply by RonnyPfannschmidt** [reply]
thanks for the follow, i'm happy the solution worked well for you
